tests/Microsoft.DotNet.UpgradeAssistant.TestHelpers/UpgradeStepExtensions.cs

```diff
@@ -18,5 +20,17 @@ public static void SetStatus(this UpgradeStep step, UpgradeStepStatus status)
 
             method.Invoke(step, new object[] { status });
         }
+
+        public static void ClearRazorDocuments(this RazorUpdaterStep step)
```

Comment:
Does this need to be in the testhelpers library? That will bring in the razorupdater dependency there.

Reply:
I can put it in the Razor step tests which might be cleaner.

Reply:
Fixed